tests/examples/sql_inject_target/factureconf.py

```diff
@@ -33,6 +33,35 @@
 # must be defined in an OrderedDict in order to retain attr write order!
 
 import collections
+import re
```

Comment:
I wonder if it makes sense to break out a new example where we specifically highlight this more advanced functionality?  Ideally I'd like to be able to point to a simple example for people to start with, and for them to discover this additional stuff by digging a bit, I think.  What do you think?

Reply:
Yeah sounds good to me 👍 I'll set up a new example!